### PR TITLE
Add missing assignment to x in create_vgg16_network

### LINKED ISSUE #134: Minor bug in VGG16 block in network.py
In block 2 of the create_vgg16_network function, line 39 should be 
`x = MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)`
instead of just 
`MaxPooling2D(pool_size=(2, 2), strides=(2, 2))(x)`
It is missing the equality.

### PR DESCRIPTION
**Pull request by ManasviKundalia**
Fixes #134.

### CODE REVIEW

**Comment by Shashi456** [comment]
@ManasviKundalia you have a conflict you need to rebase your code .

File: collect.py [hunk 1]
```diff
@@ -111,6 +111,7 @@ def get_elements_with_properties(driver, elem_properties, children):
 
 
 def do_something(driver, elem_properties=None):
+
```
Comment (marco-c) [inline collect.py]: There are several unrelated changes in this file, can you remove them?
